File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/Brand.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Brand{
    @SerializedName("id")
    @Expose
    public String Id;

    @SerializedName("brandName")
    @Expose
    public String BrandName;
}

File: CarsMatter.Infrastructure/Models/AzureTables/Brand.cs
using Microsoft.WindowsAzure.Storage.Table;
using Newtonsoft.Json;

namespace CarsMatter.Infrastructure.Models.AzureTables
{
    public class Brand : TableEntity
    {
        public string Id { get; set; }

        [JsonIgnore]
        [IgnoreProperty]
        public string HttpPath { get; set; }

        public string BrandName { get; set; }

        public Brand(string id)
        {
            this.Id = id;
            this.PartitionKey = id;
            this.RowKey = id;
        }

        public Brand()
        {
        }
    }
}
